sdk/tools/openrewrite-sample-compiler-ci.yml

```diff
@@ -0,0 +1,73 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - /sdk/tools/
```

Comment:
Do we want to trigger this for everything under sdk/tools or should we limit this to only the openrewrite packages?

Reply:
The scope can be tightened. I copied the initial pipeline format from the tools ci

Reply:
Let's keep the scope of this pipeline to only OpenRewrite directories as this is doing only OpenRewrite specific validation